dokka-subprojects/analysis-kotlin-descriptors-compiler/src/main/kotlin/org/jetbrains/dokka/analysis/kotlin/descriptors/compiler/translator/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -740,8 +740,8 @@ private class DokkaDescriptorVisitor(
     ): DFunction {
         val dri = parent.copy(callable = Callable.from(descriptor))
         val isGetter = descriptor is PropertyGetterDescriptor
-        val isExpect = descriptor.isExpect
-        val isActual = descriptor.isActual
+        val isExpect = propertyDescriptor.isExpect
```

Comment:
this is a minor fix for K1 compiler (just to make test easier).
`descriptor` of getter/setter in properties doesn't contain information about `expect/actual` (the same is true in K2, there is no such properties there).
So I've decided to make behaviour consistent. Not sure, that will affect anything rather than the model itself

Reply:
> descriptor of getter/setter in properties doesn't contain information about expect/actual 

`isExpect/isActual `always returns `false` for accessors.
What is about to keep the same behavior in K2 to avoid the change in Documentable model?

Reply:
yeah, that's also fine I believe. Do you mind if I will change it for K1 also to just use `false` instead of `isExpect/isActual` properties to make it more clear?